apps/condo/domains/ticket/schema/TicketFile.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Generated by `createschema ticket.TicketFile 'organization:Text;file?:File;ticket?:Relationship:Ticket:SET_NULL;'`
+ */
+
+const { Relationship, File } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketFile')
+const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
+
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const Adapter = new FileAdapter('ticket')
+
+// zuch(todo): find a way to upload images in jest tests
+const TicketFile = new GQLListSchema('TicketFile', {
+    schemaDoc: 'File attached to the ticket',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+        organization: ORGANIZATION_OWNED_FIELD,
+        file: {
+            schemaDoc: 'File object with meta information and publicUrl',
+            type: File,
+            adapter: Adapter,
+            isRequired: false,
+        },
+        ticket: {
+            schemaDoc: 'Link to ticket',
+            type: Relationship,
+            ref: 'Ticket.files',
+            many: false,
+            isRequired: false,
+            knexOptions: { isNotNullable: false }, // ticketFile can be without ticket on create (temporary)
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+        },
+    },
+    hooks: {
+        afterDelete: async ({ existingItem }) => {
+            if (existingItem.file) {
```

Comment:
We don’t need to delete files :)

Reply:
We need to remove temporary files that do not belong to any of the ticket

Reply:
It’s ok to keep it